24life/blocks/article-hero-video/article-hero-video.css

```diff
@@ -89,6 +89,10 @@ div.article-hero-video-wrapper {
   filter: brightness(0.80);
 }
 
+.cards-container:has(+ .newsletter-form-container){
+  text-align: center;
+}
+
```

Comment:
Please don't use `.cards` - it is not intended for this. 

`:has` is not ready to be used. https://caniuse.com/css-has

I see two options: 
1. Accept that the image will be left-aligned, remove this code and update the document to use no block. 
2. in style.css, after `main .section.no-gap` add 
```css
main .section.center {
  text-align: center;
}
```
and add section metadata like:
![Screenshot 2023-10-15 at 01 10 23](https://github.com/hlxsites/24life/assets/2250964/7f7a1cc5-d21a-4b8d-9030-8413608f69ed)

Reply:
Sure. Thank you for the feedback. Fixed. Pl advise.